integration_tests/__tests__/resolve.test.js

```diff
@@ -12,5 +12,9 @@ const runJest = require('../runJest');
 
 test('resolve platform modules', () => {
   const result = runJest('resolve');
-  expect(result.status).toBe(0);
+  const didFail = result.status !== 0;
+  if (didFail) {
+    console.log(result.stderr);
+  }
```

Comment:
This is a leftover, right? I know these integration tests don't give great error feedback, but it gets better while running inside specific directory (when debugging what's wrong)

Reply:
That was not obvious at all. Contributing document doesn't mention this :-(
I spent half an hour guessing why the test doesn't work without feedback.

As I wrote above:

>I also added some logging to the test runner for this integration test because when it failed, there used to be no indication at all as to the reasons why it was failing.

I can remove though.

Reply:
Can you open up a separate issue for this? If nothing else it should be documented, but maybe we can make it better